frontend/src/utils/tsxUtils.tsx

```diff
@@ -309,25 +295,29 @@ export class RadioOptionGroup<T> extends Component<{
     render() {
         const p = this.props;
 
-        const radioGroup = (
-            <Radio.Group className="radioOptionGroup" value={p.value} onChange={e => p.onChange(e.target.value)}>
-                {p.options.map(kv => (
-                    <Radio key={kv.key ?? kv.value} value={kv.value} disabled={p.disabled}>
-                        <div style={{ fontWeight: 500, display: 'inline-block', paddingBottom: '2px', paddingLeft: '10px', verticalAlign: 'middle' }}>{kv.title}</div>
-                        <div style={{ marginLeft: '27px', color: '#999', whiteSpace: 'normal' }}>{kv.subTitle}</div>
-                        <AnimatePresence>
-                            {kv.content && (p.showContent == 'always' || p.value == kv.value) && (
-                                <MotionDiv animProps={animProps_radioOptionGroup} key={String(kv.value)} style={{ marginLeft: '27px', marginTop: '12px' }}>
-                                    <div>{kv.content}</div>
-                                </MotionDiv>
-                            )}
-                        </AnimatePresence>
-                    </Radio>
-                ))}
-            </Radio.Group>
-        );
-
-        return radioGroup;
+        return <RadioGroup
+            direction="column"
+            // TODO - we need to make the API more TS safe and make name optional
+            name=""
+            // @ts-ignore
+            value={p.value}
+            // @ts-ignore
```

Comment:
What are the errors you were getting here?

Reply:
`T` can be null, while `RadioGroup` accepts `undefined`. Solvable problem, but I'd leave it for later.